open-api-spec.yaml

```diff
@@ -850,6 +1001,89 @@ components:
           format: date-time
           pattern: '2022-04-01T10:24:36.11Z'
           description: The date and time when the outgoing payment was updated.
+    quote:
+      title: Quote
+      description: 'A **quote** resource represents the quoted amount details with which an Outgoing Payment may be created.'
+      type: object
+      examples:
+        - id: 'https://openpayments.guide/alice/quotes/ab03296b-0c8b-4776-b94e-7ee27d868d4d'
+          accountId: 'https://openpayments.guide/alice/'
+          receivingAccount: 'https://openpayments.guide/aplusvideo/'
+          receiveAmount:
+            value: '2500'
+            assetCode: USD
+            assetScale: 2
+          sendAmount:
+            value: '2500'
+            assetCode: USD
+            assetScale: 2
+          amountType: 'fixed-receive'
+          createdAt: '2022-03-12T23:20:50.52Z'
+          updatedAt: '2022-04-01T10:24:36.11Z'
+          expiresAt: '2022-04-12T23:20:50.52Z'
+        - id: 'https://openpayments.guide/alice/quotes/8c68d3cc-0a0f-4216-98b4-4fa44a6c88cf'
+          accountId: 'https://openpayments.guide/alice/'
+          receivingPayment: 'https://openpayments.guide/aplusvideo/incoming-payments/45d495ad-b763-4882-88d7-aa14d261686e'
+          sendAmount:
+            value: '2500'
+            assetCode: USD
+            assetScale: 2
+          receiveAmount:
+            value: '2198'
+            assetCode: EUR
+            assetScale: 2
+          amountType: 'fixed-send'
+          createdAt: '2022-03-12T23:20:50.52Z'
+          updatedAt: '2022-04-01T10:24:36.11Z'
+          expiresAt: '2022-04-12T23:20:50.52Z'
+      properties:
+        id:
+          type: string
+          format: uri
+          description: The URL identifying the quote.
+          example: 'https://openpayments.guide/alice/quotes/8c68d3cc-0a0f-4216-98b4-4fa44a6c88cf'
+          readOnly: true
+        accountId:
+          type: string
+          format: uri
+          description: The URL of the account from which this quote's payment would be sent.
+          example: 'https://openpayments.guide/alice/'
+          readOnly: true
+        receivingAccount:
+          type: string
+          format: uri
+          description: The URL of the account that would be paid.
+          example: 'https://openpayments.guide/aplusvideo/'
+        receivingPayment:
+          type: string
+          format: uri
+          description: The URL of the incoming payment that would be paid.
+          example: 'https://openpayments.guide/aplusvideo/incoming-payments/45d495ad-b763-4882-88d7-aa14d261686e'
+        receiveAmount:
+          $ref: '#/components/schemas/amount'
+        sendAmount:
+          $ref: '#/components/schemas/amount'
+        amountType:
+          type: string
+          description: The type of quote amount.
+          enum:
+            - fixed-send
```

Comment:
Would a client actually need to do a fixed-send quote before authorization?
The grant can't have `receiveAmount`, so why not just create the fixed-send outgoing payment and specify the (minimum) `receiveAmount`?

Reply:
:thinking: But then the client wouldn't have a way to check if funds are available for a fixed-send outgoing payment...